docs/policy/service-changes-2018.md

```diff
@@ -0,0 +1,188 @@
+Service Changes 2018
+====================
+
+As you may have heard, the Open Science Grid (OSG) is undergoing some changes to its fabric of services.
+Some services will be retired, most services will be migrated with little planned effect on sites,
+and some services will be migrated with greater effect on sites.
+This document is intended to guide OSG Site Administrators through these changes.
+The following sections list various OSG services and whether or not site administrator action is required.
+
+!!! note
+    There will be announcements prior to retirement or migration of any OSG service.
+
+The OSG Technology team will be holding [office hours](#office-hours) for any questions or comments about these service 
```

Comment:
Users might miss the link. I suggest moving the link to the end and adding a phrase like “; see the end of this page for details”. Or something.